engine/render/src/render/font_renderer.cpp

```diff
@@ -835,8 +849,8 @@ namespace dmRender
 
         for (int line = 0; line < line_count; ++line) {
             TextLine& l = lines[line];
-            int16_t x = (int16_t)(x_offset - OffsetX(te.m_Align, l.m_Width) + 0.5f);
-            int16_t y = (int16_t) (y_offset - line * leading + 0.5f);
```

Comment:
rounded vertext coordinate is the reason of dancing text (https://github.com/defold/defold/issues/7197)